File: scripts/lua/originate.lua
-- Example Lua script to originate. luarun
freeswitch.console_log("info", "Lua in da house!!!\n");

local session = freeswitch.Session("sofia/10.0.1.100/1001");
session:execute("playback", "/sr8k.wav");
session:hangup();

File: scripts/perl/originate.pl
# Example Perl script to originate. perlrun
freeswitch::console_log("info", "Perl in da house!!!\n");

$session = new freeswitch::Session("sofia/10.0.1.100/1002") ;
$session->execute("playback", "/sr8k.wav");
$session->hangup();
